english/smores

```diff
@@ -0,0 +1 @@
+You decide to use your lighter and the marshmallows to make smores! It's too bad you have chocolate...
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.